### DISCUSSION TITLE
List of tags on DockerHub

### DISCUSSION THREAD

**Discussion by oskrabanek** [Azure/api-management-self-hosted-gateway#204] [Q&A] [answered]
I would like to automate checking the new version of the docker image and I wasn't able to do it on DockerHub as the tags for the image are not available using API nor displayed on the [page](https://hub.docker.com/_/microsoft-azure-api-management-selfhostedgateway). Would it be possible to add it there?

**Comment by tomkerkhove** [comment, marked as answer]
You can pull all image tags from https://mcr.microsoft.com/v2/azure-api-management/gateway/tags/list or use a tool like Renovate to automatically update images

**Reply by oskrabanek** [reply]
Perfect, that's what I was looking for. Thank you. Maybe mentioning it on the DockerHub would be nice.

**Reply by tomkerkhove** [reply]
It is actually already there:
![image](https://user-images.githubusercontent.com/4345663/233054247-78cf8a13-cb3e-4d8a-b340-ba2593592790.png)

**Reply by oskrabanek** [reply]
Ah, my apologies. I've missed this reference.
